### DISCUSSION TITLE
Change Column Order Error on Windows 7 x64

### DISCUSSION THREAD

**Discussion by paulp575** [LesFerch/WinSetView#79] [Q&A] [answered]
Running WinSetView version 2.76 on Windows 7 Pro (64-bit).
On the "Global" page, when I click on the icon to "Change column order" I get an error on line 995 char 3.

![Change column order script error](https://github.com/LesFerch/WinSetView/assets/44275957/c896f7b4-5dba-4438-94af-14ee1e468559)

**Comment by LesFerch** [comment]
I now see the same issue here. Previously, I was testing on a 32 bit Windows 7 VM, which works fine. For reasons yet to be determined, that Math function does not work on Windows 7 64 bit even though it's fine on Windows 10 64 bit. Weird. That function is only used once and I can easily replace it with an if statement, but it _should_ work.

I'll roll a fix into my next update.

**Comment by LesFerch** [comment, marked as answer]
This has now been fixed in the current release (Ver 2.77).

**Comment by paulp575** [comment]
Thank you for fixing it.
Since I'm moving to Win 10 next week I'll not be able to test it on Win 7.
Appreciate your efforts.
